src/providers/node/mod.rs

```diff
@@ -142,6 +145,21 @@ impl NodeProvider {
             }
         }
 
+        if Turborepo::is_turborepo(app) {
+            let turbo_cfg = Turborepo::get_config(app)?;
+            let dlx = NodeProvider::get_package_manager_dlx_command(app);
+            return if let Some(build_cmd) = Turborepo::get_build_cmd(&turbo_cfg) {
+                Ok(Some(build_cmd))
+            } else if let Some(app_name) = Turborepo::get_app_name(env) {
+                Ok(Some(format!("{} turbo run {}:build", dlx, app_name)))
+            } else if NodeProvider::has_script(app, "build")? {
+                let pkg_manager = NodeProvider::get_package_manager(app);
+                Ok(Some(format!("{} run build", pkg_manager)))
+            } else {
+                Ok(None)
+            };
```

Comment:
Can we default to the regular node build command behaviour below at this point? The logic is the same right now, but ideally we don't have to maintain two versions. So if there is no Turbo build pipeline and no build script in the Turbo app name, we fallback to regular Node provider build command.